cmd/subnetcmd/deploy.go

```diff
@@ -258,6 +260,25 @@ func controlKeysLoop(controlKeysPrompt string, network models.Network) ([]string
 				continue
 			}
 			controlKeys = append(controlKeys, controlKey)
+		case delCtrlKey:
```

Comment:
Could you modify the deploy function to just use the same code you modified in `precompiles.go`? You might need to accept an extra arg or two, but that way there would only be one code path.

Reply:
I just pushed such an attempt.